chainio/clients/avsregistry/reader_test.go

```diff
@@ -0,0 +1,158 @@
+package avsregistry
+
+import (
+	"context"
+	"log/slog"
+	"math/big"
+	"os"
+	"testing"
+
+	"github.com/Layr-Labs/eigensdk-go/logging"
+	"github.com/Layr-Labs/eigensdk-go/testutils"
+	"github.com/Layr-Labs/eigensdk-go/types"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/stretchr/testify/require"
+)
+
+func TestReaderMethods(t *testing.T) {
+	anvilStateFileName := "contracts-deployed-anvil-state.json"
```

Comment:
Think i would prefer either having this as a constant or as a part of a test config

either

```
type TestConfig struct {
	AnvilStateFileName string
	LogLevel           slog.Level
}


func getDefaultTestConfig() TestConfig {
	return TestConfig{
		AnvilStateFileName: "contracts-deployed-anvil-state.json",
		LogLevel:           slog.LevelDebug,
	}
}
```

and maybe refactoring out the test environment setup to a helper function that can be used across tests in `chainio/clients`, seeing similar setup in https://github.com/Layr-Labs/eigensdk-go/pull/361/files